main.bicep

```diff
@@ -366,12 +475,28 @@ resource sharedLogicAppIdentity 'Microsoft.ManagedIdentity/userAssignedIdentitie
   location: resourcesLocation
 }
 
+resource anomalyDetector 'Microsoft.CognitiveServices/accounts@2022-12-01' = if (deployAnomalyDetectionResources) {
+  name: 'msdyn-iiot-sdi-anomaly-detector-${uniqueIdentifier}'
+  location: resourcesLocation
+  sku: {
+    name: 'F0'
+  }
+  kind: 'AnomalyDetector'
+  properties: {
+    apiProperties: {
+      statisticsEnabled: false
```

Comment:
Can you add a comment here on why this is false?

Reply:
That is not needed, mistake, when copying templates. 
![image](https://user-images.githubusercontent.com/116812567/234211616-b5a6e1c6-a42c-4652-9a51-c22093a91f45.png)